README.md

```diff
@@ -1,6 +1,6 @@
 # hello-world
 
-Hi Humans!
+Hi Fellow Humans!
```

Comment:
I would suggest to sound more patronizing, for example "My Fellow Humans", rather than just "Fellow Humans"